dev/ToolTip/ToolTip_rs5_themeresources.xaml

```diff
@@ -2,52 +2,56 @@
 
-        <ResourceDictionary x:Key="HighContrast">
-            <StaticResource x:Key="ToolTipBorderBrush" ResourceKey="SystemControlTransientBorderBrush" />
-            <Thickness x:Key="ToolTipBorderThemePadding">8,5,8,7</Thickness>
         </ResourceDictionary>
 
         <ResourceDictionary x:Key="Light">
+            <StaticResource x:Key="ToolTipForegroundBrush" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="ToolTipBackgroundBrush" ResourceKey="AcrylicBackgroundFillColorDefaultBrush" />
             <StaticResource x:Key="ToolTipBorderBrush" ResourceKey="SurfaceStrokeColorDefaultBrush" />
-            <Thickness x:Key="ToolTipBorderThemePadding">8,5,8,7</Thickness>
+        </ResourceDictionary>
+
+        <ResourceDictionary x:Key="HighContrast">
+            <StaticResource x:Key="ToolTipForegroundBrush" ResourceKey="SystemColorWindowTextColorBrush" />
+            <StaticResource x:Key="ToolTipBackgroundBrush" ResourceKey="SystemColorWindowColorBrush" />
+            <StaticResource x:Key="ToolTipBorderBrush" ResourceKey="SystemColorWindowTextColorBrush" />
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
+    <Thickness x:Key="ToolTipBorderPadding">9,6,9,8</Thickness>
+
     <Style TargetType="ToolTip" BasedOn="{StaticResource DefaultToolTipStyle}" />
 
     <Style x:Key="DefaultToolTipStyle" TargetType="ToolTip">
-        <Setter Property="Foreground" Value="{ThemeResource ToolTipForeground}" />
-        <Setter Property="Background" Value="{ThemeResource ToolTipBackground}" />
+        <Setter Property="Foreground" Value="{ThemeResource ToolTipForegroundBrush}" />
+        <Setter Property="Background" Value="{ThemeResource ToolTipBackgroundBrush}" />
         <Setter Property="BorderBrush" Value="{ThemeResource ToolTipBorderBrush}" />
         <Setter Property="BorderThickness" Value="{ThemeResource ToolTipBorderThemeThickness}" />
         <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
         <Setter Property="FontSize" Value="{ThemeResource ToolTipContentThemeFontSize}" />
-        <Setter Property="Padding" Value="{ThemeResource ToolTipBorderThemePadding}" />
-        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" />
+        <Setter Property="Padding" Value="{ThemeResource ToolTipBorderPadding}" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="ToolTip">
                     <ContentPresenter x:Name="LayoutRoot"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         Background="{TemplateBinding Background}"
-                        BackgroundSizing="OuterBorderEdge"
+                        contract7Present:BackgroundSizing="InnerBorderEdge"
```

Comment:
Maybe make this a TemplateBinding: contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"

With setter above: <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />

Reply:
good idea